resources/assets/js/videos/videoContainer.vue

```diff
@@ -189,7 +189,7 @@ export default {
             }
 
             let promise = new Vue.Promise((resolve, reject) => {
-                this.video.addEventListener('seeked', resolve);
+                this.video.addEventListener('timeupdate', resolve);
```

Comment:
This should also still be `seeked` because it resolves the promise returned by the `seek()` method.